src/mscorlib/src/System/Runtime/ExceptionServices/ExceptionServicesCommon.cs

```diff
@@ -125,9 +125,14 @@ public Exception SourceException
         // the original thread.
         public void Throw()
         {
-            // Restore the exception dispatch details before throwing the exception.
+            throw GetThrowableException();
+        }
+
+        public Exception GetThrowableException()
```

Comment:
Public surface area?

Reply:
Eep, was internal - took it out, put it back, typo'd

Reply:
fixed